Tests/survey/test_agent_switchboard_adapter_contracts.py

```
@@ -0,0 +1,94 @@
+
+def test_versions_and_structured_process_boundary() -> None:
+    text = ADAPTER.read_text(encoding="utf-8")
+    assert 'REQUEST_VERSION = "agentswitchboard-invocation/v2"' in text
+    assert 'RESULT_VERSION = "agentswitchboard-result/v2"' in text
+    assert "subprocess.run(" in text and "timeout=args.timeout_seconds" in text
+    assert "shell=True" not in text and "Invoke-Expression" not in text
+
+
+def test_valid_native_and_bridge_fixtures() -> None:
+    with tempfile.TemporaryDirectory() as temp:
+        root=Path(temp); req=root/"request.json"; out=root/"result.json"
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)